src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/mvn.cpp

```diff
@@ -0,0 +1,337 @@
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+std::string MvnLayerCPUTest::getTestCaseName(testing::TestParamInfo<MvnLayerCPUTestParamSet> obj) {
+    basicCpuMvnParams basicParamsSet;
+    CPUSpecificParams cpuParams;
+    fusingSpecificParams fusingParams;
+    ElementType inputPrecision, outputPrecision;
+    std::tie(basicParamsSet, cpuParams, fusingParams, inputPrecision, outputPrecision) = obj.param;
+
+    InputShape inputShapes;
+    ElementType netPrecision;
+    ngraph::AxisSet axes;
+    bool acrossChanels, normalizeVariance;
+    double eps;
+    std::tie(inputShapes, netPrecision, axes, acrossChanels, normalizeVariance, eps) = basicParamsSet;
+
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::partialShape2str({inputShapes.first}) << "_";
+    result << "TS=";
+    for (const auto& shape : inputShapes.second) {
+        result << "(" << CommonTestUtils::vec2str(shape) << ")_";
+    }
+    result << "Precision=" << netPrecision << "_";
+    if (!axes.empty()) {
+        result << "ReductionAxes=" << CommonTestUtils::vec2str(axes.to_vector()) << "_";
+    } else {
+        result << "AcrossChannels=" << (acrossChanels ? "TRUE" : "FALSE") << "_";
+    }
+    result << "NormalizeVariance=" << (normalizeVariance ? "TRUE" : "FALSE") << "_";
+    result << "Epsilon=" << eps;
+    result << "_"
+           << "CNNInpPrc=" << inputPrecision;
+    result << "_"
+           << "CNNOutPrc=" << outputPrecision;
+
+    result << CPUTestsBase::getTestCaseName(cpuParams);
+
+    result << CpuTestWithFusing::getTestCaseName(fusingParams);
+
+    return result.str();
+}
+
+bool MvnLayerCPUTest::isSupportedTestCase() {
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    // "initAcrossChannels = false" is not supported by ACL for NHWC layout
+    if (!inFmts.empty() && (inFmts.front() == nwc ||
+                            inFmts.front() == nhwc ||
+                            inFmts.front() == ndhwc) &&
+        !acrossChanels) return false;
+#endif
+    return true;
+}
+
+void MvnLayerCPUTest::SetUp() {
+    targetDevice = CommonTestUtils::DEVICE_CPU;
+
+    basicCpuMvnParams basicParamsSet;
+    CPUSpecificParams cpuParams;
+    fusingSpecificParams fusingParams;
+    ElementType inPrc;
+    ElementType outPrc;
+    std::tie(basicParamsSet, cpuParams, fusingParams, inPrc, outPrc) = this->GetParam();
+
+    std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+    std::tie(postOpMgrPtr, fusedOps) = fusingParams;
+
+    InputShape inputShapes;
+    ElementType netPrecision;
+    ngraph::AxisSet axes;
+    bool normalizeVariance;
+    double eps;
+    std::tie(inputShapes, netPrecision, axes, acrossChanels, normalizeVariance, eps) = basicParamsSet;
+
+    if (!isSupportedTestCase()) {
+        GTEST_SKIP() << "Skip MVN test since such combination of parameters is not supported." << std::endl;
+    }
```

Comment:
It looks like we have some cases not supported on ARM. But, isn't it enough to simply not instance them in the dedicated tests set? Why do we need to check the condition here?
Also, if ARM have to support these configurations, don't we have to make a dedicated ticket?

Reply:
It makes common tests logic more complex, since we need to split one common test case into common test case and x64 test case (to exclude case not supported by arm).
The issue is ACL has quite a lot of limitations, so some limitations are combined in some test cases. It makes "split" logic even more complex and its maintenance would be complicated.

Reply:
Ok. Then we need a follow up ticket to refactor this solution later.